### ISSUE TITLE
Need to get information of lead buyer

### ISSUE THREAD

**Issue by guascce** [OP-TED/ted-rdf-mapping#181]
The indicators require to know who is the lead buyer. For now it is taken for granted that the first buyer in the list is lead.

**Comment by muricna** [comment]
@guascce Can I take it that you mean that for the scoreboard the first buyer in the list is considered the lead?  If we were to indicate that the first buyer is the lead in the rdf that would mean changing the semantic.  The rdf is to replicate the xml, how do you suggest we deal with this?

**Comment by guascce** [comment]
We need to discuss this with DG GROW. I believe that there is an implicit information in the first buyer of the list, that means that he is the lead, so giving this buyer the group -leader role would not change the semantic meaning. But I would prefer to get confirmation by GROW.

**Comment by muricna** [comment]
Even if it is implicit for the analysis we need to be sure that the buyers filling in the forms were aware of this.

**Comment by kvistgaard** [comment]
Indeed, no semantics shouldn't be inferred on the basis of assumption. But there is no harm in indicating a fact from the source structure, for example, that something is the first item in a list. In case the role is an instance-specific to a notice (from what I see, it is), it could be either as a new property to be used for the buyer (no need to axiomatize it, so it's reused in other cases) or as a subproperty of `epo:refersToRole`. This way  the interpretation of the first buyer on the list (lead ... then country) will be postponed and most importantly decoupled from the mapping.

However, in the sample @csnyulas pointed to, I saw some distinctions made in the way buyer URI is constructed. Fore example, in [this sample](https://github.com/OP-TED/ted-rdf-mapping/blob/main/mappings/package_F03/output/002705-2021/002705-2021.ttl), there are 47 instances of Buyer. One URI  starting with
<http://data.europa.eu/a4g/resource/Buyer/2021-S-002-002705/...>
and 46 URI starting with
<http://data.europa.eu/a4g/resource/AdditionalBuyer/2021-S-002-002705/...>

I haven't checked the mappings, but it could be that this way of constructing is related to this issue.
Currently, there is no class "Additional Buyer" in ePO.
